src/Core/ConfigurationInterface.php

```diff
@@ -23,10 +23,33 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
+
 namespace PrestaShop\PrestaShop\Core;
 
+/**
+ * Interface ConfigurationInterface defines contract for simple configuration
+ *
+ * @deprecated since 1.7.4.0. Use PrestaShop\PrestaShop\Core\Configuration\AdvancedConfigurationInterface instead.
+ */
 interface ConfigurationInterface
 {
-    public function get($key);
+    /**
+     * Get configuration value
+     *
+     * @param string $key
+     * @param mixed $default
+     *
+     * @return mixed
+     */
+    public function get($key, $default = null);
```

Comment:
you can't change definition of the interface ;)

Reply:
hmm, but `Adapter/Configuration` implements interface with default value `public function get($key, $default = null)`

Reply:
so `Configuration` class doesn't respect his own interface :/

Reply:
Why do you fix the old Interface ? It's a BC break (although indeed it might be acceptable). But isn't it enough to have `SimpleConfigurationInterface` that is correct ?